tox.ini

```diff
@@ -21,6 +21,22 @@ deps =
     isort
 commands = isort -c
 
+[testenv:mypy]
+skip_install = true
+deps =
+    -rrequirements.txt
+    mypy
+# Mypy fails if 3rd party library doesn't have type hints configured.
+# Alternative to ignoring imports would be to write custom stub files, which
+# could be done at some point.
+commands = mypy --ignore-missing-imports metadata_backend/
+
+[testenv:pydocstyle]
+skip_install = true
+deps =
+    pydocstyle
```

Comment:
if you add https://pypi.org/project/flake8-docstrings/ as a dependency at the `testev:flake8` then this does not need to be done in a separate env.

Reply:
we will have a separate stage for generating and testing documentation, so this could be included with flake8